packages/shared/src/UI/components/AssetsManagement/CollectibleCard.tsx

```diff
@@ -49,6 +58,19 @@ export interface CollectibleCardProps extends HTMLProps<HTMLDivElement> {
     disableNetworkIcon?: boolean
     /** disable inspect NFT details */
     disableInspect?: boolean
+    selectedAsset?: Web3Helper.NonFungibleAssetAll
```

Comment:
`CollectibleCard` should not reply on `selectedAsset`, do not passing `selectedAsset` to this component but an `isSelected` (or `selected`) status instead.